data/campaigns/Secrets_of_the_Ancients/scenarios/18_Abandoned_Outpost.cfg

```diff
@@ -426,7 +426,7 @@ Now the process is complete for both of us. I posted a guard so nothing could ha
 
         [message]
             speaker=Ardonna
-            message= _ "We have conquered death! Let us bring the news back to Wesnoth. We should to let them know that nobody ever need die again."
+            message= _ "We have conquered death! Let us bring the news back to Wesnoth. We should - to let them know that nobody ever need die again."
```

Comment:
It's hard to tell what the character is, but it looks like a regular hyphen to me - should be an em dash according to https://wiki.wesnoth.org/Typography_Style_Guide

Reply:
This sentence sounds awkward regardless of whether there's a dash there…

Reply:
If you want the dash you can get rid of the "we should" and merge it with the previous sentence. This edit otherwise doesn't work for me.

Reply:
Without the dash, you'd probably need to drop "to", as in "we should let"…

Reply:
How about a colon?

Reply:
Will change the text to this:
"We have conquered death! Let us bring the news back to Wesnoth! Let all in this world know that nobody ever need die again."
> ![image](https://user-images.githubusercontent.com/41433185/135223067-a3e6aa89-fa92-42eb-8adf-a8b3bac84054.png)

"ever need die" is grammatically incorrect but we will permit it for tone or archaism.

Reply:
> "ever need die" is grammatically incorrect but we will permit it for tone or archaism.

How is it grammatically incorrect? The phrasing "nobody need die again" is grammatically valid; at most the placement of "ever" may be strange. It doesn't feel wrong like this, but it might fit better if it were placed between "need" and "die".

> How about a colon?

Ehhh, I dunno. A colon is almost never correct in speech, as far as I know. There's probably some corner cases where it applies, but I'm not sure this is one of them…

Reply:
> How is it grammatically incorrect?

Hmm, yeah. I just said it was grammatically incorrrect because - Grammarly, ugh. I try not to rely on it and only use for QA reasons, at the end of everything, and with a grain of salt. Anyway, you can see the screenshot where it was underlined red so you see why I ended up saying it was incorrect.

Reply:
It's something of an unusual construction, so it wouldn't surprise me if the grammar checker doesn't understand it. But that's also part of the reason to keep it here, for the archaic flavour (it was likely more common in the past).